src/main/java/com/redhat/cloud/notifications/routers/EndpointService.java

```diff
@@ -98,10 +98,9 @@
             count = resources.getEndpointsCount(principal.getAccount());
         }
 
-        return Uni.combine().all().unis(
-                endpoints.collectItems().asList(),
-                count
-        ).asTuple().onItem().transform(t -> new EndpointPage(t.getItem1(), new HashMap<>(), new Meta(t.getItem2())));
+        return endpoints.collectItems().asList()
```

Comment:
For information - collectItems is deprecated, use collect instead

Reply:
We're still using an old Mutiny version where `collect` is not available, but this will change pretty soon.